instrumentation/github.com/gorilla/mux/otelmux/config.go

```diff
@@ -86,6 +88,22 @@ func WithSpanNameFormatter(fn func(routeName string, r *http.Request) string) Op
 	})
 }
 
+// WithSpanStartOption applies options to all the HTTP span created by the
+// instrumentation.
+func WithSpanStartOption(o ...oteltrace.SpanStartOption) Option {
```

Comment:
In [`otelhttp`](https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp#WithSpanOptions) and [`otelgrpc`](https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc#WithSpanOptions) this option is called `WithSpanOptions`. Can we name it in the same way for sake of consistency?

Reply:
Absolutely. I'll leave the code as is for now, since this may not be relevant, if it is replaced with the option to add stacktraces instead - see [this comment](https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5251#issuecomment-2036544502).
I'll return to this, if we decide to keep the function.